tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 project = detect_secrets
 # These should match the travis env list
-envlist = py27,py35,py36,pypy
+envlist = py27,py35,py36,py37,pypy
```

Comment:
Could you add a `skip_missing_interpreters = true` under this, in case some developer env's don't have `3.7`